docs/source/models_and_datasets.md

```diff
@@ -17,4 +17,15 @@ limitations under the License.
 
 # Morpheus Models
 
-Morpheus comes with a number of pretrained models with corresponding training, validation scripts, and datasets. The latest release of these models can be found [here](https://github.com/nv-morpheus/Morpheus/blob/-/models).
\ No newline at end of file
+Morpheus comes with a number of pretrained models with corresponding training, validation scripts, and datasets. The latest release of these models can be found [here](https://github.com/nv-morpheus/Morpheus/blob/-/models).
+
+|Model|GPU Mem Req|Description|
+|-----|-----------|-----------|
+|[Anomalous Behavior Profiling (ABP)](https://github.com/nv-morpheus/Morpheus/blob/-/models#anomalous-behavior-profiling-abp)|2015MiB|This model is an example of a binary classifier to differentiate between anomalous GPU behavior such as crypto mining / GPU malware, and non-anomalous GPU-based workflows (e.g., ML/DL training). The model is an XGBoost model.|
+|[Digital Fingerprinting (DFP)](https://github.com/nv-morpheus/Morpheus/blob/-/models#digital-fingerprinting-dfp)|4.97MiB|This use case is currently implemented to detect changes in users' behavior that indicate a change from a human to a machine or a machine to a human. The model is an ensemble of an Autoencoder and fast Fourier transform reconstruction.|
+|[Fraud Detection](https://github.com/nv-morpheus/Morpheus/blob/-/models#fraud-detection)|76.55MiB|This model shows an application of a graph neural network for fraud detection in a credit card transaction graph. A transaction dataset that includes three types of nodes, transaction, client, and merchant nodes is used for modeling. A combination of GraphSAGE along XGBoost is used to identify frauds in the transaction networks.|
+|[Flexible Log Parsing](https://github.com/nv-morpheus/Morpheus/blob/-/models#flexible-log-parsing)|1612MiB|This model is an example of using Named Entity Recognition (NER) for log parsing, specifically apache web logs.|
+|[Phishing Email Detection](https://github.com/nv-morpheus/Morpheus/blob/-/models#phishing-email-detection)|1564MiB|Phishing email detection is a binary classifier differentiating between phishing/spam and non-phishing/spam emails and SMS messages.|
+|[Ransomware Detection Model](https://github.com/nv-morpheus/Morpheus/blob/-/models#ransomware-detection-via-appshield)|n/a|This model shows an application of DOCA AppShield to use data from volatile memory to classify processes as ransomware or bengin. This model uses a sliding window over time and feeds derived data into a random forest classifiers of various lengths depending on the amount of data collected.|
```

Comment:
```suggestion
|[Ransomware Detection Model](https://github.com/nv-morpheus/Morpheus/blob/-/models#ransomware-detection-via-appshield)|n/a|This model shows an application of [DOCA App Shield](https://docs.nvidia.com/doca/sdk/app-shield-programming-guide/index.html) to use data from volatile memory to classify processes as ransomware or benign. This model uses a sliding window over time and feeds derived data into a random forest classifiers of various lengths depending on the amount of data collected.|
```

Reply:
I had a question about this sentence:
"This model uses a sliding window over time and feeds derived data into a random forest classifiers of various lengths depending on the amount of data collected."

The "into a" implies a single random forest classifier, but "classifiers of various lengths" implies its plural, not sure which is correct.